storage/engine/db.cc

```diff
@@ -63,8 +63,8 @@ namespace {
 // in storage/engine/keys.go.
 const rocksdb::Slice kKeyLocalRangeIDPrefix("\x00\x00\x00i", 4);
 const rocksdb::Slice kKeyLocalRangeKeyPrefix("\x00\x00\x00k", 4);
-const rocksdb::Slice kKeyLocalResponseCacheSuffix("res-");
-const rocksdb::Slice kKeyLocalTransactionSuffix("txn-");
+const rocksdb::Slice kKeyLocalResponseCacheSuffix("\x00res-", 5);
+const rocksdb::Slice kKeyLocalTransactionSuffix("\x00\x01txn-", 6);
```

Comment:
Response cache keys are created using `MakeRangeIDKey` which uses `EncodeInt` for the middle portion of the key. Transaction keys are created using `MakeRangeKey` which uses `EncodeBytes` for the middle portion of the key. I'm probably going to have to change this when I change response cache keys to use `EncodeVarUint64`.